recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,9 @@
 pytorch_version:
-  - 1.8.0
-  - 1.9.0
-  - 1.9.1
-  - 1.10.0
+  - 1.11.0
+  - 1.12.0
+numpy:
+  - 1.19
```

Comment:
This is not supported. You have to use the global conda-forge pinnings.
```suggestion
```

Reply:
@mikemhenry added that to prevent it from explicitly requesting a version that's incompatible with the PyTorch it's installing.  (See https://github.com/conda-forge/openmm-torch-feedstock/pull/23#issuecomment-1194545671.)  What's the correct fix?

Reply:
@jaimergp maybe there is something else wrong with this feedstock, but it kept trying to build with numpy 1.16 which is why I set the NumPy version to something newer

Reply:
This is an [example](https://github.com/conda-forge/freud-feedstock/blob/79ecb8d2af74909de199a9bce359f2931dd2e09c/recipe/meta.yaml) of how it should look like. I guess you tried that, so we'll need to figure that out. What we can't do is to override the global pinnings -- that would reduce the compatibility of this package.

Reply:
Is it possible the real problem is in the pytorch feedstock?  Shouldn't it automatically give a numpy version that's compatible with the pytorch version?

Reply:
Is it ok for us to specify numpy >= 1.19?  Somehow we need to force it to install a version that's compatible with PyTorch.